lib/Interface/core/ImageContainer.cpp

```diff
@@ -23,55 +23,92 @@
 
 #include <cassert>
 
-// Image Constructor from OpenCV Mat.
```

Comment:
Any specific reason for removing this comment?

Reply:
Thanks to point it out, there is no need to remove it and I will add it back.